bin/xbps-uhelper/xbps-uhelper.1

```diff
@@ -0,0 +1,130 @@
+.Dd July 10, 2020
+.Dt XBPS-UHELPER 1
+.Os
+.Sh NAME
+.Nm xbps-uhelper
+.Nd Utility to extract and compare package strings.
+.Sh SYNOPSIS
+.Nm
+.Op OPTIONS
+.Ar ACTION
+.Op ARGUMENTS
+.Sh DESCRIPTION
+.Nm
+provides a number of functions to extract and compare package strings.
+.Sh OPTIONS
+.Bl -tag -width -x
+.It Fl C Ar dir
+Path to xbps.conf file.
+.It Fl d
+Output debugging messages to stderr.
+.It Fl r Ar rootdir
+Use
+.Ar rootdir
+as rootdir.
+.It Fl V
+Prints the XBPS release version.
+.El
+.Sh ACTIONS
+.Bl -tag -width -x
+.It arch
+Return the XBPS native architecture.
+.It binpkgarch Ar binpkg
+Return architecture of package file
+.Ar binpkg .
+.It binpkgver Ar binpkg
+Return version of package file
+.Ar binpkg .
+.It cmpver Ar instver reqver
+Return success if installed package version
+.Ar instver
+matches the required version
+.Ar reqver ,
+failure otherwise.
```

Comment:
Actually 1, 0, 255 for first >, =, < than second. This is useful to check what xbps thinks about versions containing letters.

Reply:
Good catch, fixed.